projects/demo-components/src/fullscreen-map-layout.jsx

```diff
@@ -10,7 +10,7 @@ const FullscreenMapLayout = ({ headerProps, mapComponent, children }) => {
       <div className='flex-child-grow relative flex flex--column'>
         {mapComponent}
         {/* medium+ sidebar */}
-        <div className='absolute left mx6 my6 px6 py6 top-ml w360 none block-ml z4 bottom mb30 hmax-full'>
+        <div className={`${sidebarSize} absolute left mx6 my6 px6 py6 top-ml none block-ml z4 bottom mb30 hmax-full`}>
```

Comment:
Recommend using `classnames` for this sort of thing instead of template literals which can get cluttered.